app/pubspec.lock

```diff
@@ -604,4 +590,4 @@ packages:
     source: hosted
     version: "2.1.13"
 sdks:
-  dart: ">=2.0.0-dev <=2.0.0-dev.15.0"
+  dart: ">=2.0.0-dev <=2.0.0-dev.17.0"
```

Comment:
We'll need to update the base image in Dockerfile too, then, right?

Reply:
I think that can be independent of this upper bound constraint. I had an issue with dev.17 running a local instance, not yet sure what was causing it.